map/download-preplanned-map/README.metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+    "category": "Map",
+    "description": "Take a map offline using a preplanned map area.",
+    "ignore": false,
+    "images": [
+        "DownloadPreplannedMap.png"
+    ],
+    "keywords": [
+        "map area",
+        "offline",
+        "preplanned",
+        "pre-planned"
+    ],
+    "redirect_from": "/java/latest/sample-code/download-preplanned-map.htm",
+    "relevant_apis": [
+        "DownloadPreplannedOfflineMapJob",
+        "DownloadPreplannedOfflineMapParameters",
+        "DownloadPreplannedOfflineMapResult",
+        "OfflineMapTask",
+        "PreplannedMapArea"
+    ],
+    "snippets": [
+        "src/main/java/com/esri/samples/download_preplanned_map/DownloadPreplannedMapController.java",
```

Comment:
I think that the `DownloadPreplannedOfflineMapJobListCell` should also be in this snippet list?